monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -2703,6 +2701,26 @@ abstract class Observable[+A] extends Serializable { self =>
     new OnErrorRetryCountedObservable(self, maxRetries)
   }
 
+  /** Returns an Observable that mirrors the behavior of the source,
+    * unless the source is terminated with an `onError`, in which case
+    * it tries subscribing to the source again after the `initialDelay` has
+    * expired, in the hope that it will complete without an error.
+    *
+    * The number of retries is limited by the specified `maxRetries`
+    * parameter, while also implementing an exponential backoff strategy meaning
+    * each successive retry will wait twice as long as the previous before attempting
+    * to subscribe to the source again.
+    *
+    * So for an Observable that always ends in error the total number of subscriptions that will eventually
+    * happen is `maxRetries + 1`, and the total time taken before the source fails will be
+    * `(initialDelay ^ maxAttempts - 1) / 2`, so if the `initialDelay` is 2 seconds with a `maxAttempts`
+    * value of 10, the the total time taken, assuming all retries fail, will be 512 seconds.
+    */
+  final def onErrorRetryWithBackoff(maxRetries: Long, initialDelay: FiniteDuration): Observable[A] = {
```

Comment:
I suggest `onErrorRestartWithBackoff` for consistency with other `onErrorRestartXXX` operators

Reply:
Will update accordingly :-).